### DISCUSSION TITLE
Requirements

### DISCUSSION THREAD

**Discussion by Lautaro22** [alicevision/Meshroom#1808] [Q&A]
hi! I have a AMD Radeon HD 6450 card and I want to know if I can use meshroom anyway? 
I have this error message in the "DepthMap" step:
[10:26:33.519280][error] cudaGetDeviceCount failed: CUDA driver version is insufficient for CUDA runtime version
and
[10:26:33.519280][error] This program needs a CUDA-Enabled GPU (with at least compute capability 2.0).
Thank you for your reply!

**Comment by natowi** [comment]
See https://github.com/alicevision/Meshroom/wiki/Draft-Meshing

**Reply by Lautaro22** [reply]
It work! thank!

**Reply by Lautaro22** [reply]
Hi, I have other problem. When I use the Monstree dataset the program works ok but when I put my photographs don't do the last step "texturin". And I don't know why, because don't appear error message.

**Reply by Lautaro22** [reply]
another thing is that it doesn't show me anything in the "structure for motion" step

**Reply by natowi** [reply]
Double click on the node (Texturing) to load the model in the 3d viewer
If you do not get any result your images are not enough or the model is unsuitable for photogrammetry. Take a look on some of the tutorials listed here: https://github.com/alicevision/Meshroom/wiki/Tutorials
